icons/Chevron.jsx

```diff
@@ -15,18 +15,34 @@ export default function Arrow ({ color, styles, className, ...props }) {
       {...props}>
       <path
         className={classNames('illustration__stroke')}
-        d='M6.5,9.3l4,4l4,-4'
+        d={d}
         strokeLinecap='round' />
     </svg>
   )
 }
 
-Arrow.defaultProps = {
+Chevron.defaultProps = {
   color: 'blue',
   styles: {}
 }
 
-Arrow.propTypes = {
+Chevron.propTypes = {
   color: PropTypes.oneOf(colors),
   styles: PropTypes.object
 }
+
+// The geometrical center would be starting at x = 8, y = 6, but because of
+// https://medium.com/@erqiudao/the-play-button-is-not-optical-alignment-4cea11bda175#.3p6p4iry8
+// it should be 1px off
+
+export const Right = (props) =>
```

Comment:
It would be nice to add display names for this with a more fully qualified name. Currently the output in the showroom is `<Up />`, `<Down />` and so on. I'd prefer having `<Chevron.Up />` and so on.